### PR TITLE
Add Stafi related repositories

### PR DESCRIPTION
**Pull request by Tore19**
Add more repositories for Stafi.

Many thanks.

### CODE REVIEW

**Comment by jubos** [comment]
Thanks for the PR.  If you want to add Stafi as a subecosystem you need a make a separate toml file for it and then reference it in the polkadot.toml.

**Comment by jubos** [comment]
If you have the rust toolchain installed, you can also run "make validate" to ensure the dataset is valid before uploading your changes.

**Comment by Tore19** [comment]
@jubos Hi, Just changed as you said. If there is any problem, please let me know. Thanks!

**Comment by jubos** [comment]
@Tore19, thanks for the changes.  One other fix is that stafiprotocol is actually a github user and not an organization.  You can either remove it or upgrade stafiprotocol to an org on github.

File: data/ecosystems/s/stafi.toml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Ecosystem Level Information
+title = "Stafi"
+
+sub_ecosystems = []
+
+github_organizations = ["https://github.com/stafiprotocol"]
```
Comment (jubos) [inline data/ecosystems/s/stafi.toml:6]: This is just a github user as opposed to an org, so you can make this an empty array for now.

Comment (jubos) [inline data/ecosystems/s/stafi.toml:6]: This is just a github user as opposed to an org, so you can make this an empty array for now.

**Review by jubos — CHANGES_REQUESTED** [review]
Just need to tweak the github_organizations array.

File: data/ecosystems/s/stafi.toml (hunk 1 above)
Comment (Tore19) [reply data/ecosystems/s/stafi.toml]: Done! Thanks!

**Pull request merged**